cloudconfig/podcfg/image.go

```diff
@@ -62,21 +61,25 @@ func RebuildOldOperatorImagePath(imagePath string, ver version.Number) string {
 	return tagImagePath(imagePath, ver)
 }
 
-func tagImagePath(path string, ver version.Number) string {
+func tagImagePath(fullPath string, ver version.Number) string {
+	fullPathParts := strings.Split(fullPath, "/")
```

Comment:
Lets just remove all this string bashing and instead use github.com/distribution/distribution/reference to Parse image references.

Reply:
done